docs/sphinx/releases/whats_new_1_5_0.rst

```diff
@@ -8,17 +9,560 @@
+* We added serialization support for ``std::variant`` and ``std::tuple``.
+* ``result_of`` and ``is_callable`` are now deprecated and replaced by
+  ``invoke_result`` and ``is_invocable`` to conform to C++20.
+* We continued with the modularization, it is now easier to enable/disable the
+  distributed runtime option. An important amount of headers have been
+  deprecated. We adapted the namespaces and headers we could to be closer to the
+  standard ones (:ref:`api <_public_api>`).
+* It is now possible to have a basic CUDA support including a helper function to
+  get a future from a CUDA stream and target handling. They are available under
+  the ``hpx::cuda::experimental`` namespace and they can be enabled with the
+  ``HPX_WITH_ASYNC_CUDA=ON`` cmake option.
+* We added a new ``hpx::mpi::experimental`` namespace for getting futures from
+  an asynchronous MPI call and a new minimal MPI executor (add the name).
+* A polymorphic executor has been implemented to reduce compile times as a
+  function accepting executor can potentially be instanciated only once instead
+  of multiple times with different executors. It accepts the function signature
+  as a template argument. It needs to be constructed from any other executor.
+  Please note, that the function signatures that can be scheduled using
+  ``then_execute``, ``bulk_sync_execute``, ``bulk_async_execute`` and
+  ``bulk_then_execute`` are slightly different.
+* The underlying executor of ``block_executor`` has been updated to a newer one.
+* We have added a parameter to ``auto_chunk_size`` to control the amount of
+  iterations to measure.
+* We have added performance counters type information (aggregating,
+  monotonically increasing, average count, average timer, etc.).
+* The tasks are now re-scheduled on the same worker thread they were suspended
+  on to avoid cache misses from moving from one thread to the other. This
+  behavior doesn't prevent the task from being stolen.
+* We have added a new configuration option ``hpx.exception_verbosity`` to allow
+  to control the level of verbosity of the exceptions (3 levels available).
+* ``broadcast_to``, ``broadcast_from``, ``scatter_to`` and ``scatter_from`` have
+  been added to the collectives, modernization of ``gather_here`` and
+  ``gather_there`` with futures taken by rvalue references. See the breaking
+  change on ``all_to_all`` in the next section. All the collectives do not need
+  supporting macros anymore (e.g. specifying the data types used for a
+  collective operation using ``HPX_REGISTER_ALLGATHER`` and similar are not
+  needed anymore).
+* A new API function to get the number of cores which are idle
+  (``hpx::get_idle_core_count``) and another one returning a bitmask
+  representing the currently idle cores (``get_idle_core_mask``).
+* We have added an experimental option to only enable the local runtime, you can
+  enable the distributed runtime with ``HPX_WITH_DISTRIBUTED_RUNTIME=ON`` (or
```

Comment:
I'd maybe turn this around and say "you can disable the distributed runtime at compile time with `HPX_WITH_DISTRIBUTED_RUNTIME=OFF` (defaults to `ON`)"?